### PR TITLE
feat(home page): add section two to home page

### LINKED ISSUE #21: homePagesectiontwo
**Task title**
homepagesectiontwo

**Task description**
add section two to the home page 

**Subtasks**
divide the home page  section two into three man sections 

**Screenshots**
Put at least one screenshot here

**Link to the component on Figma**
Put the link here

### PR DESCRIPTION
**Pull request by oualidelh**
#21

# Related Issue

- Resolve #21

### CODE REVIEW

File: src/components/homepage/SectionTwo.jsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                    <p className='md:w-[50%] w-[80%] text-center text-sm text-gray-400 font-normal'>
+                        {t("globalGolas.discription")}
+                    </p>
+                </div>
+                <div className='grid lg:grid-cols-6 md:grid-cols-4 grid-cols-2 justify-items-center justify-self-center gap-2 w-full'>
+                    {globalGoalsImages.map((goal, index) => (
+                        <Link
+                            className=' flex w-[60%] h-auto justify-center '
+                            href={`${goal.title}`}
+                            key={index}
```
Comment (HachemBouhadede) [inline src/components/homepage/SectionTwo.jsx:129]: well done, next time try using something else other then the index as keu

File: src/components/homepage/SectionTwo.jsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+                                {user.FeedBack}
+                            </p>
+                        </div>
+                    ))}
+                </div>
+            </div>
+            {/* events mages section */}
+            <div className='flex md:flex-row flex-col  px-4 gap-3 mt-16 md:px-8'>
+                <div className='flex flex-col  gap-y-3'>
+                    <div className='h-[40%] rounded-md overflow-hidden shadow-md'>
```
Comment (HachemBouhadede) [inline src/components/homepage/SectionTwo.jsx:173]: well on relative dimension

**Review by HachemBouhadede — APPROVED** [review]
welld one ! reviewed and ready to be pushed, any furthur imporvment can be done in future issues <3

**Review by Khalil-NOUI — APPROVED** [review]
good job.

**Pull request merged**